processor/redactionprocessor/README.md

```diff
@@ -119,6 +123,11 @@ part of the value is masked with a fixed length of asterisks.
 `blocked_key_patterns` applies to the values of the keys matching one of the patterns.
 The value is then masked according to the configuration.
 
+`hash_function` defines the function for hashing the values (or substrings of values)
```

Comment:
```suggestion
`hash_function` defines the function for hashing values of matched keys or matches in values
```
The only way to hash a substring is to hash the matched part of a value, right? I think we should make the wording a bit clearer as to what the input to the hash functions is.